### ISSUE TITLE
[semi] Doesn't seem to apply to class members

### ISSUE THREAD

**Issue by strutcode** [typescript-eslint/typescript-eslint#407]
I want to disallow semicolons globally in my Typescript project, but I noticed that the rule isn't being applied to class members. See the following code:

```ts
import { FreeCamera } from 'babylonjs'

export default class PanCamera extends FreeCamera {
  public invertX: boolean = false // valid
  public invertY: boolean = false; // also valid

  public pan(deltaX: number, deltaY: number): void {
    this.position.x += deltaX * (this.invertX ? -1 : 1) // valid
    this.position.y += deltaY * (this.invertY ? -1 : 1); // invalid
  }
}
```

This is my `.eslintrc`:
```json
{
  "parser": "@typescript-eslint/parser",
  "parserOptions": {
    "project": "./tsconfig.json"
  },
  "plugins": [
    "@typescript-eslint"
  ],
  "extends": [
    "airbnb-base",
    "plugin:@typescript-eslint/recommended"
  ],
  "env": {
    "browser": true,
    "node": true,
    "es6": true,
    "worker": true
  },
  "rules": {
    "semi": ["error", "never"],
    "brace-style": ["error", "stroustrup"],
    "@typescript-eslint/indent": ["error", 2],
    "no-new": 0,
    "lines-between-class-members": ["error", "always", {
      "exceptAfterSingleLine": true
    }]
  }
}
```

And my versions:
```json
  "devDependencies": {
    // ...
    "@typescript-eslint/eslint-plugin": "^1.6.0",
    "@typescript-eslint/parser": "^1.6.0",
    "eslint": "5.3.0",
    "eslint-config-airbnb-base": "13.1.0",
    "eslint-plugin-import": "^2.14.0",
    "typescript": "^3.4.1",
    // ...
  },
```
